ydb/tools/olap_workload/__main__.py

```diff
@@ -21,55 +22,142 @@ def __init__(self, endpoint, database, duration):
-                    PRIMARY KEY(id)
-                    )
-                    PARTITION BY HASH(id)
-                    WITH (
-                        STORE = COLUMN
-                    )
+        try:
+            self.session.execute_scheme(
                 """
-                    % table_name
+                CREATE TABLE %s (
+                id Int64 NOT NULL,
+                i64Val Int64,
+                PRIMARY KEY(id)
+                )
+                PARTITION BY HASH(id)
+                WITH (
+                    STORE = COLUMN
                 )
+            """
+                % table_name
+            )
 
-                print("Table %s created" % table_name)
-            except ydb.SchemeError as e:
-                print(e)
+            print("Table %s created" % table_name)
+        except Exception as e:
+            print(type(e), e)
 
     def drop_table(self, table_name):
-        with self.pool.checkout() as s:
-            try:
-                s.drop_table(self.database + "/" + table_name)
-
-                print("Table %s dropped" % table_name)
-            except ydb.SchemeError as e:
-                print(e)
+        print(f"Drop table {table_name}")
+        try:
+            self.session.drop_table(self.database + "/" + table_name)
+        except Exception as e:
+            print(type(e), e)
+
+    def add_column(self, table_name, col_name):
+        print(f"Add column {table_name}.{col_name}")
+        try:
+            q = f"ALTER TABLE {table_name} ADD COLUMN {col_name} Int64"
+            print(q)
+            self.session.execute_scheme(q)
+        except Exception as e:
+            print(type(e), e)
+
+    def drop_column(self, table_name, col_name):
+        print(f"Drop column {table_name}.{col_name}")
+        try:
+            self.session.execute_scheme(f"ALTER TABLE {table_name} DROP COLUMN {col_name}")
+        except Exception as e:
+            print(type(e), e)
+
+
+    def add_batch(self, table_name, schema):
+        print(f"Add batch {table_name}")
+
+        schema = list(schema) + ["id"]
+        column_types = ydb.BulkUpsertColumns()
+
+        for c in schema:
+            column_types.add_column(c, ydb.PrimitiveType.Int64)
+
+        data = []
+        
+        for i in range(10):
+            data.append({c: random.randint(0, 1000000) for c in schema})
+
+        self.driver.table_client.bulk_upsert(self.database + "/" + table_name, data, column_types)
+
+    def list_tables(self):
+        db = self.driver.scheme_client.list_directory(self.database)
+        return [t.name for t in db.children if t.type == ydb.SchemeEntryType.COLUMN_TABLE]
+
+    def list_columns(self, table_name):
+        path = self.database + "/" + table_name
+        return [c.name for c in self.session.describe_table(path).columns]
+    
+    def rows_count(self, table_name):
+        return self.driver.table_client.scan_query(f"SELECT count(*) FROM {table_name}").next().result_set.rows[0][0]
```

Comment:
it is better to use scan_query from session object

Reply:
+ for better readability you can use update query to `SELECT count(*) as count from {table_name}` to use rows[0].count